tests/test_datasets/test_dataset.py

```diff
@@ -838,6 +838,7 @@ def test_update_regressors_with_futuremixin_transform(ts_with_regressors, transf
     _test_update_regressors_fit_transform(deepcopy(ts_with_regressors), deepcopy(transforms), expected_regressors)
 
 
+@pytest.mark.xfail(reason="TSDataset 2.0: blocked by another transforms")
```

Comment:
You can remove this two tests at all, we don't need them in TSDataset anymore because now transforms are responsible for updating regressors